ac.soton.coda.vhdl.feature/feature.xml

```diff
@@ -20,7 +17,7 @@
       provider-name="%featureVendor"
       plugin="ac.soton.coda.vhdl.branding"
       license-feature="ac.soton.coda.licence"
-      license-feature-version="1.1.1.qualifier">
+      license-feature-version="1.1.2.release">
 
```

Comment:
I still have licence version = 1.1.1.qualifier.
 If we release it will go to 1.1.1.release?